frontend/src/app/testResults/TestResultsList.tsx

```diff
@@ -520,6 +525,27 @@ export const DetachedTestResultsList = ({
                   defaultValue={getDateOrEmptyString(filterParams.endDate)}
                 />
               </div>
+              <Select
+                label="Condition"
+                name="disease"
+                value={filterParams.disease ?? ""}
+                options={[
+                  {
+                    value: MULTIPLEX_DISEASES.COVID_19,
+                    label: MULTIPLEX_DISEASES.COVID_19,
+                  },
+                  {
+                    value: MULTIPLEX_DISEASES.FLU_A,
+                    label: MULTIPLEX_DISEASES.FLU_A,
+                  },
+                  {
+                    value: MULTIPLEX_DISEASES.FLU_B,
+                    label: MULTIPLEX_DISEASES.FLU_B,
+                  },
+                ]}
+                defaultSelect
+                onChange={setFilterParams("disease")}
+              />
```

Comment:
Per story AC, the values here should be loaded from the backend so we dont have to keep adding more values as we add more diseases, doesnt have to block this PR, but it could be added as a followup pr as part of the same ticket

Reply:
I'm honestly not sure if I'm sold on pulling those values from the backend here to begin with. Adding new diseases to SimpleReport will already involve touchpoints in the frontend code, like [our constants](https://github.com/CDCgov/prime-simplereport/blob/fbdd80caa304a9d959b22c4f5a3d08731eb17877/frontend/src/app/testResults/constants.ts#L1-L5).

Also, `HIV` and `RSV` have already been added to the database but we're not ready to expose those values to the disease filter, so we'd need to turn around and hardcode _those_ values somewhere instead.

Basically I'm leaning towards there being a logical distinction between `diseases that SR "knows about"` and `diseases that users can submit for`, at least at this time.

Whatchy'all think?

Reply:
Hmm that's a good point where the frontend already requires some hardcoded reference for the diseases. At our next pairing session I'm curious to get your thoughts on this as it relates to what I'm working on with adding HIV support since it seems like we'll always need some way to distinguish which AOE questions to show